app/bin/tools/notify_service.dart

```diff
@@ -22,16 +24,18 @@ Future main(List<String> args) async {
   }
 
   await withProdServices(() async {
-    registerNotificationClient(new NotificationClient());
+    registerAnalyzerClient(
+        new AnalyzerClient(activeConfiguration.analyzerServicePrefix));
+    registerSearchClient(new SearchClient());
     final String service = args[0];
     if (service == 'analyzer' && args.length == 3) {
-      await notificationClient.notifyAnalyzer(
-          args[1], args[2], new Set<String>());
+      await analyzerClient.triggerAnalysis(args[1], args[2], new Set<String>());
     } else if (service == 'search' && args.length == 2) {
-      await notificationClient.notifySearch(args[1]);
+      await searchClient.triggerReindex(args[1]);
     } else {
       _printHelp();
     }
-    await notificationClient.close();
+    await analyzerClient.close();
+    await searchClient.close();
```

Comment:
Normally it is encouraged to use `registerScopeExitCallback(analyzerClient.close)` after `registerAnalyzerClient` instead of doing it yourself, since that will do it also in the case of exceptions and does it precisely when leaving the service scope.

But since this is a tool, it's fine.